tomviz/PipelineModel.cxx

```diff
@@ -769,6 +771,22 @@ void PipelineModel::operatorAdded(Operator* op)
     }
     endInsertRows();
   }
+
+  // If there is a transformed data output to move, move it to be a child
+  // of this operator
+  if (transformedDataSource) {
+    auto transformedIndex = this->dataSourceIndex(transformedDataSource);
+    auto transformedDataSourceItem = this->treeItem(transformedIndex);
+    auto oldParent = transformedDataSourceItem->parent()->op();
+    auto oldParentIndex = this->operatorIndex(oldParent);
+
+    beginMoveRows(oldParentIndex, transformedIndex.row(), transformedIndex.row(), operatorIndex,
+                  operatorTreeItem->childCount());
+    transformedDataSourceItem = transformedDataSourceItem->detach();
+    operatorTreeItem->attach(transformedDataSourceItem);
+    endMoveRows();
```

Comment:
This is a duplicate of the dataSourceMoved(...) method, can we refactor so we only have it in one place?